db/db_impl_secondary.h

```diff
@@ -182,10 +192,15 @@ class DBImplSecondary : public DBImpl {
   // method can take long time due to all the I/O and CPU costs.
   Status TryCatchUpWithPrimary() override;
 
+
+  // try to find log reader using log_number from log_readers_ map, initialize
+  // if it doesn't exist
   Status MaybeInitLogReader(uint64_t log_number,
                             log::FragmentBufferedReader** log_reader);
 
  protected:
+  // ColumnFamilyCollector is a special write batch handler which does nothing
```

Comment:
Maybe remove 'special' because it is actually no different from other write batch handlers?